lib/PostEditorBlocks/WooCommerceBlocksIntegration.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace MailPoet\PostEditorBlocks;
+
+use Automattic\WooCommerce\Blocks\Domain\Services\ExtendRestApi;
+use Automattic\WooCommerce\Blocks\Package;
+use Automattic\WooCommerce\Blocks\StoreApi\Schemas\CheckoutSchema;
+use MailPoet\Config\Env;
+use MailPoet\Models\Subscriber;
+use MailPoet\Settings\SettingsController;
+use MailPoet\WooCommerce\Subscription as WooCommerceSubscription;
+use MailPoet\WP\Functions as WPFunctions;
+
+// @todo Disable if blocks is inactive
```

Comment:
Is there a task to finish this `@todo`? I don't want this to be here forever. Could this not being implemented cause any issues?